openerp/addons/base/res/res_partner.py

```diff
@@ -538,6 +552,17 @@ def write(self, vals):
             self._fields_sync(partner, vals)
         return result
 
+    #Gravatar Image
+    def _get_gravatar_image(self, email):
+        email_hash = hashlib.md5(email.lower()).hexdigest()
+        url = "https://www.gravatar.com/avatar/" + email_hash
+        try:
+            image_content = urllib2.urlopen(url + "?d=404&s=128", timeout=5).read()
+            gravatar_image = base64.b64encode(image_content)
+            return gravatar_image
+        except Exception:
+            _logger.info("Error/Request timeout in Gravatar discovery")
```

Comment:
None will be returned; I think returning False is better here (default value for void fields).